### PR TITLE
PT-152482389 Push created spend_tx into the mempool

### PR DESCRIPTION
**Pull request by zp-sd**
* [Finishes #152482389]

### CODE REVIEW

File: apps/aecore/src/aec_next_nonce.erl [hunk 1]
```diff
@@ -0,0 +1,17 @@
+-module(aec_next_nonce).
+
+-export([for_account/0]).
+
+for_account() ->
```
Comment (sennui) [inline apps/aecore/src/aec_next_nonce.erl]: this will need to talk to code in mempool PR

Comment (zp-sd) [reply apps/aecore/src/aec_next_nonce.erl]: Agreed.

File: apps/aehttp/src/aehttp_dispatch_ext.erl [hunk 2]
```diff
@@ -113,28 +113,26 @@ handle_request('GetAccountBalance', Req, _Context) ->
...
                                #{sender => SenderPubkey,
                                  recipient => RecipientPubkey,
                                  amount => Amount,
                                  fee => Fee,
                                  nonce => Nonce}),
-            %% TODO: sign and push to the mempool
+            {ok, SignedTx} = aec_keys:sign(SpendTx),
+            ok = aec_tx_pool:push(SignedTx),
+            %% TODO: send to peers via external API
+            %% To be done when POST /tx is implemented
```
Comment (zp-sd) [inline apps/aehttp/src/aehttp_dispatch_ext.erl]: /cc @uwiger

**Comment by zp-sd** [comment]
@lucafavatella I believe this should be covered in `PT-152481063 Build block proposal`, which is in works right now.

**Comment by lucafavatella** [comment]
> @lucafavatella I believe this should be covered in PT-152481063 Build block proposal, which is in works right now.

Ok. From the code I see [txs picked from mempool](https://github.com/aeternity/epoch/blob/c98342e717fb076dcd8a72cb61d67ac1580860ad/apps/aecore/src/aec_mining.erl#L73) and [then](https://github.com/aeternity/epoch/blob/c98342e717fb076dcd8a72cb61d67ac1580860ad/apps/aecore/src/aec_blocks.erl#L86) checked and [processed](https://github.com/aeternity/epoch/blob/c98342e717fb076dcd8a72cb61d67ac1580860ad/apps/aecore/src/txs/aec_tx.erl#L93) - by calling the [`aec_spend_tx` module](https://github.com/aeternity/epoch/blob/c98342e717fb076dcd8a72cb61d67ac1580860ad/apps/aecore/src/txs/aec_spend_tx.erl).

The only thing I am not sure about is height field of account.

**Comment by zp-sd** [comment]
@lucafavatella Ticket filed `PT-152678722 Investigate whether storing block height is needed in state strees`.

**Comment by lucafavatella** [comment]
> Ticket filed PT-152678722 Investigate whether storing block height is needed in state strees.

Thanks.

**Pull request merged**